google/google-how-to/gce/set-hostname-using-cloudinit.rst

```diff
@@ -23,42 +32,53 @@ Here INSTANCE_NAME is your VM name and the metadata is specified using a KEY=VAL
 For more details about this, see `Set Hostname`_ in the cloud-init documentation. 
 
 
-Additional change needed for Ubuntu 24.04 and later
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Changes based on new defaults (Ubuntu 24.04+)
+---------------------------------------------
 
-In GCE images that use Ubuntu 24.04 (or later), the ``/etc/hostname`` file is no longer present by default, and the cloud-init key ``create_hostname_file`` is set to false. This implies that whenever a user or tool (such as cloud-init) tries to set the hostname using the underlying ``hostname`` command, it will fail and the hostname will be overwritten by Google's DHCP service. 
+In GCE images that use Ubuntu 24.04 or later, the ``/etc/hostname`` file is no longer present by default, and the cloud-init key ``create_hostname_file`` is set to false. 
 
-To avoid this failure, you'll need to set ``create_hostname_file`` to true in the user-data file (apart from specifying the required hostname):
+Implications for using cloud-init
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Due to the way the underlying ``hostname`` command works, whenever a user or tool (such as cloud-init) tries to set the hostname on a system without ``/etc/hostname``, it will only be set transiently and will be overwritten by Google's DHCP service. To avoid this, you'll need to set ``create_hostname_file`` to true in the user-data file:
 
 .. code::
 
```

Comment:
Again, would recommend including `#cloud-config` at the top of the userdata file